src/main/scala/org/constellation/gossip/sampling/RandomPeerSampling.scala

```diff
@@ -11,10 +15,29 @@ import scala.util.Random
 /**
   * Peer sampling strategy based on uniform random sample selection
   */
-class RandomPeerSampling[F[_]: Concurrent](selfId: Id, cluster: Cluster[F]) extends PeerSampling[F, Id] {
-
-  def selectPaths(fanout: Int): F[Seq[IndexedSeq[Id]]] =
-    getPeers.map(randomPath(_, fanout))
+class RandomPeerSampling[F[_]](selfId: Id, cluster: Cluster[F])(implicit F: Concurrent[F]) extends PeerSampling[F] {
+
+  private val logger: SelfAwareStructuredLogger[F] = Slf4jLogger.getLogger[F]
+  private val fanout: Int = 1
```

Comment:
are we still using fanout?

Reply:
Nope. But as you can see it is `RandomPeerSampling.scala` but we use `PartitionerPeerSampling.scala`. We can technically remove `RandomPeerSampling.scala` or leave it in repo. It is just a leftover because I needed any implementation of `PeerSampling`